Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -41,12 +41,17 @@ public void PreProc_InvalidArgs()
         }
 
         [TestMethod]
-        public void PreProc_LocalPropertiesOverrideServerSettings()
+        public void PreProc_EndToEnd_SuccessCase()
```

Comment:
This is a new test that is a generalised version of the old "PreProc_LocalPropertiesOverrideServerSettings". The new version is more of an overall sanity check of the pre-processor flow and behaviour.